client/src/components/DetailedCard/DetailedCardForm.tsx

```diff
@@ -0,0 +1,103 @@
+  return (
+    <Box>
+      <Grid container xs={12} className={classes.container} direction="row">
+        <Grid container item xs={9} direction="column">
+          <Box className={classes.itemBox}>
+            <Typography className={classes.boxContent}>
+              <div>
+                <MenuBook className={classes.cardIcons} />
+              </div>
+              Description:
+            </Typography>
+            <Box className={classes.boxItems}>
+              <textarea
+                name="description"
+                rows={3}
+                cols={57}
+                className={classes.textArea}
+                placeholder="Write a description"
+              />
+              <Box color="cornflowerblue" className={classes.buttonBox}>
+                <Button variant="contained" className={classes.descriptionButton}>
+                  Save
+                </Button>
+                <CloseIcon />
+              </Box>
+            </Box>
+          </Box>
+          <Box className={classes.deadlineBox}>
+            <Typography className={classes.boxContent}>
+              <div>
+                <AccessTime className={classes.cardIcons} />
+              </div>
+              Deadline:
+            </Typography>
+            <Box className={classes.boxItems}>
+              <Router>
+                <Link to="#">March 10</Link>
+              </Router>
+            </Box>
+          </Box>
+          <Box className={classes.itemBox}>
+            <Typography className={classes.boxContent}>
+              <div>
+                <ChatBubbleOutline className={classes.cardIcons} />
+              </div>
+              Add comment:
+            </Typography>
+            <Box className={classes.boxItems}>
+              <textarea name="comments" rows={3} cols={57} className={classes.textArea} placeholder="Write a comment" />
+              <Box color="cornflowerblue" className={classes.buttonBox}>
+                <Button variant="contained" className={classes.descriptionButton}>
+                  Save
+                </Button>
+                <CloseIcon />
+              </Box>
+            </Box>
+          </Box>
+        </Grid>
+        <Grid item xs={1}></Grid>
+        <Grid container item xs={2}>
+          <Grid item className={classes.actionGrid} container direction="column">
+            <Typography> ADD TO CARD:</Typography>
+            <Box className={classes.addCardBox}>
+              {addCard.map((item, key) => {
+                return (
+                  <Button key={key} variant="contained">
+                    {item}
+                  </Button>
+                );
+              })}
+            </Box>
+          </Grid>
+          <Grid item className={classes.actionGrid} container direction="column">
+            <Typography> ACTIONS:</Typography>
+            <Box className={classes.actionBox}>
+              {actionCard.map((item, key) => {
```

Comment:
Avoid using `index` as `key`, please.

Reply:
If I don't give a key then it shows an error in the console.
Should I leave it like that or there is some other way?

Reply:
So, this is actually a unique case here. And in this case, it's a performance concern for using index as key. Reason is, these actions alter the list of cards in columns. This can lead to some weird behaviour. 

In cases where the list will be static and not changed at all, we can use index as the key. But in this case, we want to use something like `nanoid` or `uuid`